src/test/java/lotto/domain/StatisticsTest.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@DisplayName("도메인 Statistics 테스트")
+class StatisticsTest {
+
+    @DisplayName("수익률 산출 테스트")
+    @Test
+    void getRateOfReturn() {
+        Statistics statistics = new Statistics(List.of(Statistic.THREE, Statistic.FOUR), new Budget(14000));
+        float result = statistics.getRateOfReturn();
+        assertThat(result).isEqualTo(55000 / 14000);
```

Comment:
```suggestion
        assertThat(result).isEqualTo(결과값);
```

테스트 코드에 수식을 넣는것 보다는 "결과는 이것이다" 표현하는게 더 좋다고 생각하는데 어떻게 생각하시나요?